Resources/frontend/js/jquery.adyen-payment-selection.js

```diff
@@ -352,6 +352,15 @@
 
             return "undefined" !== typeof paymentMethod.details;
         },
+        /**
+         * @param  {object} paymentMethod
+         * @return {boolean}
+         * @private
+         */
+        __enableStoreDetails: function (paymentMethod) {
```

Comment:
@acampos1916 code base is modified to take 'scheme' for storing the card details.
(left a comment in code, to make sure we remember we did this explicitly)